docs/syntax-reference.rst

```diff
@@ -956,29 +956,30 @@ Scalar functions
 +------------------------+------------------------------------------------------------+---------------------------------------------------+
 | Function               | Example                                                    | Description                                       |
 +========================+============================================================+===================================================+
-| ABS                    | ``ABS(col1)``                                              | The absolute value of a value                     |
+| ABS                    | ``ABS(col1)``                                              | The absolute value of a value.                    |
 +------------------------+------------------------------------------------------------+---------------------------------------------------+
 | ARRAYCONTAINS          |  ``ARRAYCONTAINS('[1, 2, 3]', 3)``                         | Given JSON or AVRO array checks if a search       |
 |                        |                                                            | value contains in it.                             |
 +------------------------+------------------------------------------------------------+---------------------------------------------------+
-| CEIL                   |  ``CEIL(col1)``                                            | The ceiling of a value                            |
+| CEIL                   |  ``CEIL(col1)``                                            | The ceiling of a value.                           |
 +------------------------+------------------------------------------------------------+---------------------------------------------------+
-| CONCAT                 |  ``CONCAT(col1, '_hello')``                                | Concatenate two strings                           |
+| CONCAT                 |  ``CONCAT(col1, '_hello')``                                | Concatenate two strings.                          |
 +------------------------+------------------------------------------------------------+---------------------------------------------------+
 | EXTRACTJSONFIELD       |  ``EXTRACTJSONFIELD(message, '$.log.cloud')``              | Given a string column in JSON format, extract     |
-|                        |                                                            | the field that matches                            |
+|                        |                                                            | the field that matches.  However, in many cases   |
```

Comment:
If you have a column that contains a JSON string you would still need to use EXTRACTJSONFIELD. E.g.:

`{"foo": \"{\"biz\": \"baz\"}\"}`

Its only the case where the column is really an object but declared as a string that this would work for. E.g.

`{"foo": {"biz": "baz"}}`

Maybe we should call that out more specifically?

Reply:
Good point, @rodesai . I updated the PR.